src/index.ts

```diff
@@ -0,0 +1,69 @@
+import {lintProject} from "./linter/linter.js";
+import type {LintResult} from "./linter/LinterContext.js";
+
+export type {LintResult} from "./linter/LinterContext.js";
+export type {UI5LintConfigType} from "./utils/ConfigManager.js";
+
+// Define a separate interface for the Node API as there could be some differences
+// in the options and behavior compared to LinterOptions internal type.
+export interface UI5LinerOptions {
+	/**
+	 * List of patterns to lint.
+	 */
+	filePatterns?: string[];
+	/**
+	 * Pattern/files that will be ignored during linting.
+	 */
+	ignorePatterns?: string[];
+	/**
+	 * Provides complementary information for each finding, if available
+	 * @default false
+	 */
+	details?: boolean;
+	/**
+	 * Path to a ui5lint.config.(cjs|mjs|js) file
+	 */
+	config?: string;
+	/**
+	 * Whether to skip loading of the ui5lint.config.(cjs|mjs|js) config file
+	 * @default false
+	 */
+	noConfig?: boolean;
```

Comment:
did we discuss / decide on this option vs. `config?: string | boolean` to suppress the default config?

Reply:
Nope, but here are my 2 cents on that:
As we are trying to be aligned with ESLint, there it's a [`--no-config-lookup`](https://eslint.org/docs/latest/use/command-line-interface#options) option which does excatly the same thing.

For the sake of discussion, here are the options we have listed:
1. boolean | string (default / true: "ui5lint.config.{js,mjs,cjs}"; false should suppress reading the default config file)
2. string (default: "ui5lint.config.{js,mjs,cjs}"; empty string should suppress reading the default config file)
3. separate noConfig option to suppress reading the default config file

Let's see and stick with one of these options.

Reply:
Actually, we discussed (and apparently agreed on) yet another option in CPOUI5FOUNDATION-885 😅 

> - **config: (null | string (default: "ui5lint.config.{js,mjs,cjs}"; null should suppress reading the default config file, a string overwrites the default path)**
> 
>     - _Alternatives discussed (decision above):_
>        - boolean | string (default / true: "ui5lint.config.{js,mjs,cjs}"; false should suppress reading the default config file)
>        - string (default: "ui5lint.config.{js,mjs,cjs}"; empty string should suppress reading the default config file)
>        - separate noConfig option to suppress reading the default config file

If there's a need, we can of course discuss it again tomorrow!

Reply:
As discussed today, we'll go with the current approach- a `noConfig` oprions